src/au3wrap/internal/domaccessor.cpp

```diff
@@ -27,10 +26,10 @@ Au3WaveTrack* DomAccessor::findWaveTrack(Au3Project& prj, const Au3TrackId& au3t
     return dynamic_cast<Au3WaveTrack*>(findTrack(prj, au3trackId));
 }
 
-std::shared_ptr<Au3WaveClip> DomAccessor::findWaveClip(Au3WaveTrack* track, uint64_t au3ClipId)
+std::shared_ptr<Au3WaveClip> DomAccessor::findWaveClip(Au3WaveTrack* track, int64_t au3ClipId)
 {
     for (const std::shared_ptr<Au3WaveClip>& interval : track->Intervals()) {
```

Comment:
for (const auto& interval : track->Intervals()) {

Reply:
why do you think it's better to use auto? it's not clear what type we're dealing with when we review or read the code